CMakeLists.txt

```diff
@@ -148,6 +148,12 @@ if (NOT TARGET benchmark)
   add_subdirectory(third_party/benchmark)
 endif()
 
+if (CMAKE_BUILD_TYPE MATCHES Debug)
+  message("-- DEBUG: COPY OPENMP RUNTIME LIBRARY.")
```

Comment:
You could use `message(VERBOSE)` or `message(DEBUG)` so as not to pollute the log and still have debugging statements when you change the verbosity.